src/Giveaway.js

```diff
@@ -408,32 +414,39 @@ class Giveaway extends EventEmitter {
     async roll(winnerCount = this.winnerCount) {
         if (!this.message) return [];
         // Pick the winner
-        const reactions = this.message.reactions.cache;
-        const reaction =
-            reactions.find((r) => r.emoji.name === Discord.Util.resolvePartialEmoji(this.reaction)?.name) ||
-            reactions.get(Discord.Util.resolvePartialEmoji(this.reaction)?.id);
-        if (!reaction) return [];
+        if (!this.manager.libraryIsEris) {
+            const reactions = this.message.reactions.cache;
+            var reaction =
+                reactions.find((r) => r.emoji.name === Discord.Util.resolvePartialEmoji(this.reaction)?.name) ||
+                reactions.get(Discord.Util.resolvePartialEmoji(this.reaction)?.id);
+            if (!reaction) return [];
+        } else {
+            var reactionUsers = await this.message.getReaction(this.reaction);
+            if (!reactionUsers.length) return [];
+        }
```

Comment:
Should we maybe refrain from using var?
I doesn't make any problems, but its just maybe not "best practice"?
Originally I just made it like that so I don't have to make 2 let's on top, since 1 variable name for both doesn't make sense, at least to me.
Or are there any other solutions?

Reply:
Just use const I guess..??

Reply:
Cant use outside of the if, or hdym?